cmd/metrics-scraper/app/metricsscraper.go

```diff
@@ -3,18 +3,19 @@ package app
 import (
 	"context"
 	"fmt"
-	"github.com/gin-gonic/gin"
-	"github.com/karmada-io/dashboard/cmd/api/app/router"
-	"github.com/karmada-io/dashboard/cmd/web/app/options"
+	"github.com/karmada-io/dashboard/cmd/api/app/options"
```

Comment:
you can make a copy of api/app/options pkg and reimport, because there will be difference between api options and metrics-scraper options